pyas2lib/as2.py

```diff
@@ -375,7 +382,7 @@ def build(self, data, filename=None, subject='AS2 Message',
 
         if self.receiver.compress:
             self.compressed = True
-            compressed_message = email_message.Message()
+            compressed_message = email_message.Message(policy=policy.HTTP)
```

Comment:
why do we need to set the policy on this line as well?

Reply:
True, not needed.